### ISSUE TITLE
Always showing badge 

### ISSUE THREAD

**Issue by hmbenhaim** [bharathraj-e/g_recaptcha_v3#3]
Hi,
first thanks for your plugin!
I have an issue that at first if I added `GRecaptchaV3.hideBadge()` It will no show the badge but after I refresh the app is showing the page again,
thank you1

**Comment by bharathraj-e** [comment]
````html
<style>
.grecaptcha-badge { visibility: hidden; }
</style>
````

Add this style in web/index.html - head tag

this will hide the badge initially when recaptcha js loads.

Also follow this [recaptcha faq](https://developers.google.com/recaptcha/docs/faq#id-like-to-hide-the-recaptcha-badge.-what-is-allowed)

However `GRecaptchaV3.hideBadge()` & `GRecaptchaV3.showBadge()` will work after the flutter ui got rendered

**Comment by hmbenhaim** [comment]
> ````html
> <style>
> .grecaptcha-badge { visibility: hidden; }
> </style>
> ````
> 
> Add this style in web/index.html - head tag
> 
> this will hide the badge initially when recaptcha js loads.
> 
> Also follow this [recaptcha faq](https://developers.google.com/recaptcha/docs/faq#id-like-to-hide-the-recaptcha-badge.-what-is-allowed)
> 
> However `GRecaptchaV3.hideBadge()` & `GRecaptchaV3.showBadge()` will work after the flutter ui got rendered
> 

Thank you 🙏, I'll try that.

**Comment by hmbenhaim** [comment]
It worked! Than you!

**Comment by richardloxley** [comment]
In case any one else is struggling with this: I couldn't hide the badge at all even following the instructions above, but found the solution.

Eventually I discovered that I needed to move the call to GRecaptchaV3.hideBadge() to just before I instantiated my MaterialApp - within the build method of the widget called by runApp().  Previously I'd tried calling it just after the call to await GRecaptchaV3.ready() in main().

I assume the Flutter environment hadn't yet been initialised fully at that point?

Anyway, hopefully that will help anyone else struggling!
